From 3a611df5b15dcf820bc66c093941c5f58e13d731 Mon Sep 17 00:00:00 2001
From: Cole Robinson <crobinso@redhat.com>
Date: Tue, 11 Nov 2025 09:10:03 -0500
Subject: [PATCH 1/4] selinux: Match remember/recall arguments for
 SavedStateLabel

virSecuritySELinuxSetSavedStateLabel uses remember=false, but
virSecuritySELinuxRestoreSavedStateLabel uses recall=true.

This doesn't cause problems in practice, just some redundant xattr
calls. But Set and Restore calls should be matched here.

Reviewed-by: Michal Privoznik <mprivozn@redhat.com>
Signed-off-by: Cole Robinson <crobinso@redhat.com>
---
 src/security/security_selinux.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/security/security_selinux.c b/src/security/security_selinux.c
index fa5d1568eb..19e550460c 100644
--- a/src/security/security_selinux.c
+++ b/src/security/security_selinux.c
@@ -2631,7 +2631,7 @@ virSecuritySELinuxRestoreSavedStateLabel(virSecurityManager *mgr,
     if (!secdef || !secdef->relabel)
         return 0;
 
-    return virSecuritySELinuxRestoreFileLabel(mgr, savefile, true);
+    return virSecuritySELinuxRestoreFileLabel(mgr, savefile, false);
 }
 
 
-- 
2.51.1

